web/satellite/src/components/account/billing/monthlySummary/MonthlyBillingSummary.vue

```diff
@@ -8,32 +8,26 @@
                 <h1 class="current-month-area__header__month-info__title">Current Month</h1>
                 <h2 class="current-month-area__header__month-info__title-info">{{currentPeriod}}</h2>
             </div>
-            <div class="current-month-area__header__usage-info" v-if="false">
-                <span class="current-month-area__header__usage-info__data">Usage <b class="current-month-area__header__usage-info__data__bold-text">$12.44</b></span>
-                <VButton
-                    label="Earn Credits"
-                    width="153px"
-                    height="48px"
-                />
-            </div>
         </div>
         <div class="current-month-area__content">
             <h2 class="current-month-area__content__title">DETAILED SUMMARY</h2>
             <div class="current-month-area__content__usage-charges" @click="toggleUsageChargesPopup">
                 <div class="current-month-area__content__usage-charges__head">
                     <div class="current-month-area__content__usage-charges__head__name">
-                        <svg class="current-month-area__content__usage-charges__head__name__image" v-if="!areUsageChargesShown" width="8" height="14" viewBox="0 0 8 14" fill="none" xmlns="http://www.w3.org/2000/svg">
-                            <path fill-rule="evenodd" clip-rule="evenodd" d="M0.328889 13.6272C-0.10963 13.1302 -0.10963 12.3243 0.328889 11.8273L4.58792 7L0.328889 2.17268C-0.10963 1.67565 -0.10963 0.869804 0.328889 0.372774C0.767408 -0.124258 1.47839 -0.124258 1.91691 0.372774L7.76396 7L1.91691 13.6272C1.47839 14.1243 0.767409 14.1243 0.328889 13.6272Z" fill="#2683FF"/>
-                        </svg>
-                        <svg class="current-month-area__content__usage-charges__head__name__image" v-if="areUsageChargesShown" width="14" height="8" viewBox="0 0 14 8" fill="none" xmlns="http://www.w3.org/2000/svg">
-                            <path fill-rule="evenodd" clip-rule="evenodd" d="M0.372773 0.338888C0.869804 -0.112963 1.67565 -0.112963 2.17268 0.338888L7 4.72741L11.8273 0.338888C12.3243 -0.112963 13.1302 -0.112963 13.6272 0.338888C14.1243 0.790739 14.1243 1.52333 13.6272 1.97519L7 8L0.372773 1.97519C-0.124258 1.52333 -0.124258 0.790739 0.372773 0.338888Z" fill="#2683FF"/>
-                        </svg>
-                        <span>Usage Charges</span>
+                        <div class="current-month-area__content__usage-charges__head__name__image-container">
+                            <svg class="current-month-area__content__usage-charges__head__name__image" v-if="!areUsageChargesShown" width="8" height="14" viewBox="0 0 8 14" fill="none" xmlns="http://www.w3.org/2000/svg">
```

Comment:
should be class="current-month-area__content__usage-charges__head__name__image-container__image" for styles to be applied